app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def user; end
+end
```

Comment:
- [ ] 📌 Hi there! Your UsersController is looking good, but let's make sure it follows best practices and project guidelines. For this controller, it's recommended to use `index ` and `show`  as the action names, like this:

```
class UsersController < ApplicationController
  def index; end
  def show; end
end

```


- [ ] This naming convention will help keep your code organized and align with the project requirements. Also, don't forget to update your views to match these action names. Keep up the good work! 😊👍